source/user/project.rst

```diff
@@ -174,18 +174,39 @@ privileges.
 | Member  | Members can only view the list of the project members.                    |
 +---------+---------------------------------------------------------------------------+
 
-To change the role of a project member, click the *gear* button at the end of the
-user row, and then click *Change Role*. It will enable the role dropdown.
-Choose a new role and click the *Submit* button to apply the new role to the user, or
+To change the role of a project member, choose a new role from the dropdown and
+click the *Submit* button to apply the new role to the user, or
 use the *Cancel* button to cancel the action.
 
 .. figure:: project/managerole.png
   :alt: Manage Role of a User
-  
+
   Manage Role of a User
 
 .. _manage-users:
 
+Manage User SU Budgets in a Chameleon Project
+-------------------------------
+
+Every user within a Chameleon project can have a service unit (SU) budget allocated to them.
```

Comment:
We should have a sentence explaining why a PI might want to set budgets, something like:

> Without an SU budget, there are no restrictions on how many SUs a user can use from your allocation, meaning a single user could potentially use all 20,000 SUs.